spark/src/main/java/org/apache/iceberg/spark/SparkUtil.java

```diff
@@ -20,21 +20,38 @@
 package org.apache.iceberg.spark;
 
 import java.util.List;
+import java.util.Map;
 import java.util.function.BiFunction;
 import java.util.function.Function;
 import java.util.stream.Collectors;
 import org.apache.iceberg.PartitionField;
 import org.apache.iceberg.PartitionSpec;
+import org.apache.iceberg.Schema;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.hadoop.HadoopFileIO;
 import org.apache.iceberg.io.FileIO;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.transforms.Transform;
 import org.apache.iceberg.transforms.UnknownTransform;
+import org.apache.iceberg.types.TypeUtil;
+import org.apache.iceberg.types.Types;
 import org.apache.iceberg.util.Pair;
+import org.apache.spark.sql.RuntimeConfig;
 import org.apache.spark.util.SerializableConfiguration;
 
 public class SparkUtil {
+
+  public static final String HANDLE_TIMESTAMP_WITHOUT_TIMEZONE_SESSION_PROPERTY =
+          "spark.sql.iceberg.convert-timestamp-without-timezone";
+  public static final String TIMESTAMP_WITHOUT_TIMEZONE_ERROR = String.format("Cannot handle timestamp without" +
+          " timezone fields in Spark. Spark does not natively support this type but if you would like to handle all" +
+          " timestamps as timestamp with timezone set '%s' to true. This will not change the underlying values stored" +
+          " but will change their displayed values in Spark. For more information please see" +
+          " https://docs.databricks.com/spark/latest/dataframes-datasets/dates-timestamps.html#ansi-sql-and" +
+          "-spark-sql-timestamps", HANDLE_TIMESTAMP_WITHOUT_TIMEZONE_SESSION_PROPERTY);
+  public static final String READ_TIMESTAMP_AS_TIMESTAMP_WITHOUT_TIMEZONE =
```

Comment:
I still think we only really need one property, I think we should just use "handle" for both reading and writing

Reply:
we have 1 property which responsible for reading and writing `HANDLE_TIMESTAMP_WITHOUT_TIMEZONE_SESSION_PROPERTY `

this property `READ_TIMESTAMP_AS_TIMESTAMP_WITHOUT_TIMEZONE` is responsible for converting spark `Timestamp` type to iceberg type. Depends on this property we will have `Types.TimestampType.withZone()` or `Types.TimestampType.withoutZone()`

Reply:
I'm not sure I follow based on the names, we have one property for reading and writing to existing tables, and you would like there to be an additional property for creating new tables which defaults the iceberg table to TS without Timezone?

If so can we just make those names a little clearer?

Reply:
Yeah, you are right.
`HANDLE_TIMESTAMP_WITHOUT_TIMEZONE` is responsible for allowing reading and writing of timestamp types without timezone.

`USE_TIMESTAMP_WITHOUT_TIME_ZONE_IN_NEW_TABLES` is responsible for selecting which type will be used in new tables `Types.TimestampType#withoutZone()` or `Types.TimestampType#withZone()`

I changed variable names, looks more clear now from my point of view, but please let me know if you want to change names to something else